conformance_pack/redshift.sp

```diff
@@ -299,6 +314,45 @@ query "redshift_cluster_enhanced_vpc_routing_enabled" {
   EOQ
 }
 
+query "redshift_cluster_no_default_admin_name" {
+  sql = <<-EOQ
+    select
+      arn as resource,
+      case
+        when master_username = 'awsuser' then 'alarm'
+        else 'ok'
+      end status,
+      case
+        when master_username = 'awsuser' then title || ' using default master user name.'
+        else title || ' not using default master user name.'
+      end reason
+
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
+    from
+      aws_redshift_cluster;
+  EOQ
+}
+
+query "redshift_cluster_audit_logging_enabled" {
+  sql = <<-EOQ
+    select
+      'arn:aws:redshift:' || region || ':' || account_id || ':' || 'cluster' || ':' || cluster_identifier as resource,
```

Comment:
This arn format will fail for non commercial regions since the partition values would be `aws-us-gov` or `aws-cn`.